### PR TITLE
Added Python (from Curtis Poe) and Golang

### PR DESCRIPTION
**Pull request by jhannah**
I think this Golang is equivalent?

### CODE REVIEW

File: golang.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package main
+
+type User struct {
+	name string
+	age  int
+}
+
+func (u User) execute() User {
+	return User{name: u.name, age: u.age}
```
Comment (rawleyfowler) [inline golang.go]: This should return a hashmap `map[string]any{name: u.name, age: u.age}`

**Review by rawleyfowler — COMMENTED** [review]
You should be able to add these to the `benchmark.pl` file so they can be run with the current setup!

**Comment by jhannah** [comment]
@rawleyfowler Feedback implemented?

**Review by rawleyfowler — APPROVED** [review]

**Comment by rawleyfowler** [comment]
Thanks!

**Pull request merged**